src/sentry/integrations/jira/notify_action.py

```diff
@@ -5,34 +5,109 @@
 from django import forms
 
 from sentry.rules.actions.base import IntegrationEventAction
+from sentry.shared_integrations.exceptions import IntegrationError
 
 logger = logging.getLogger("sentry.rules")
 
 
 class JiraNotifyServiceForm(forms.Form):
-    # TODO 1.0 Add form fields.
+    jira_integration = forms.ChoiceField(choices=(), widget=forms.Select())
+    assignee = forms.ChoiceField(choices=(), widget=forms.Select())
+    components = forms.ChoiceField(choices=(), widget=forms.Select())
+    description = forms.ChoiceField(choices=(), widget=forms.Select())
+    duedate = forms.ChoiceField(choices=(), widget=forms.Select())
+    fixVersions = forms.ChoiceField(choices=(), widget=forms.Select())
+    issuetype = forms.ChoiceField(choices=(), widget=forms.Select())
+    labels = forms.ChoiceField(choices=(), widget=forms.Select())
+    priority = forms.ChoiceField(choices=(), widget=forms.Select())
+    project = forms.ChoiceField(choices=(), widget=forms.Select())
+    reporter = forms.ChoiceField(choices=(), widget=forms.Select())
```

Comment:
@mgaeta Do we need to set up form fields this way? Can we use the config from `get_create_issue_config` instead?